openfisca_us/parameters/hhs/ccdf/age_limit.yaml

```diff
@@ -0,0 +1,10 @@
+description: Age limit for CCDF by state
+
+NY:
```

Comment:
Looks like this can be federal, per [this Congressional Research Service report](https://www.everycrsreport.com/reports/RL31605.html):
>Federal law defines a child as eligible for CCDF services if they are under 13 years of age and their family's income does not exceed 85% of the SMI for a family of the same size.